locksmith/src/controllers/metadataController.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-shadow */
```

Comment:
In the future we should examine our eslint config and make sure we don't need to do ignores.